polyfill/test/zoneddatetime.mjs

```diff
@@ -1473,6 +2124,263 @@ describe('ZonedDateTime', () => {
     });
   });
 
+  describe('math around DST', () => {
```

Comment:
Given that `since` and `until` will have different behavior around DST transitions, some (all?) of the added tests below probably be cloned or refactored to test `since` too.

Reply:
Ditto, I don't think I'll get this done tonight either, but let's include this in the issue to add more test coverage.